### ISSUE TITLE
Webhook listen does not update to new url

### ISSUE THREAD

**Issue by prosperoking** [PaystackOSS/paystack-cli#3]
Old ngrok url still used even after new listen command is executed

**Comment by waptik** [comment]
I don't think you need to use ngrok for that. Just use your web address on your local machine(eg: http://localhost:3000) and you're done.
Check this out https://www.youtube.com/watch?v=DaZpkJk5MK8
